.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -0,0 +1,131 @@
+name: Create cherry pick PR
+
+on:
+  push:
+    branches:
+      - orta-twoslash-repros
+  workflow_dispatch:
```

Comment:
The existing bot-triggered GHA workflows use `repository_dispatch`, but I'm like 99% sure that they are only accidentally working, given my editor says:

![image](https://github.com/microsoft/TypeScript/assets/5341706/53921ba6-65b0-4aa5-997d-5f317de9eac3)

Reply:
Eh, it's just supposed to be an array. But, these two have different formats which is quite annoying.